pkg/config/loader_test.go

```diff
@@ -86,7 +89,7 @@ func TestDefaultResources(t *testing.T) {
 	}
 
 	// Check that specifying one resource results in one resource
-	blob = `{"Resources": "Pods"}`
+	blob = `{"Resources": ["Pods"]}`
```

Comment:
Changing an old test but I feel pretty strongly on this. It seems to me this behavior would have been added just because this is how `viper` handled it. I see no reason to be be flexible on the data type. The `Resources` is a list, so the value should always be a list or fail parsing.